Source/WebCore/accessibility/AccessibilityObject.cpp

```diff
@@ -2433,6 +2434,17 @@ bool AccessibilityObject::isModalNode() const
     return false;
 }
 
+static int zIndexFromRenderLayer(RenderLayer* renderLayer)
+{
+    for (CheckedPtr layer = renderLayer; layer; layer = layer->parent()) {
+        int zIndex = layer->zIndex();
+        if (zIndex)
+            return zIndex;
+    }
```

Comment:
I think this can be shortened to:

```cpp
for (CheckedPtr layer = renderLayer; layer; layer = layer->parent()) {
    if (int zIndex = layer->zIndex())
        return zIndex;
}
```